cin_validator/cin_validator_class.py

```diff
@@ -32,10 +43,21 @@ def process_data(filename, as_dict=False):
 
 
 class CinValidationSession:
+    """A class to contain the process of CIN validation, including producing error reports,
+    and formatting error reports into JSONs for the FE.
+
+    :param data_files: Data files for validation, either a DtaContainerWrapper object, or a
+        dicitonary of DataFrames.
+    :param directory ruleset: The directory containing the validation rules to be run in a particular
+        validation session.
+    :param str issue_id: ID of individual errors to be selected for viewing using
+        select_by_id method (Error IDs, not rule codes).
+    """
+
     def __init__(
         self, data_files=None, ruleset="rules.cin2022_23", issue_id=None
     ) -> None:
-
+        """Constructor method"""
```

Comment:
"Constructor method" says what it is not what it does or why it exists. For brevity, it can be used in multiple places but there should be one init docstring that explains why an init exists.